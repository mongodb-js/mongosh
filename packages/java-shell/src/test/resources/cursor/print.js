// before
db.coll.drop();
db.coll.insertOne({a: 1});
db.coll.insertOne({a: 2});
db.coll.insertOne({a: 3});
// command
db.coll.find().forEach(d => print(d));
1
// command
db.coll.find().forEach(d => print("my document", d));
// command
console.log("Hello, world!")
// clear
db.coll.drop();
