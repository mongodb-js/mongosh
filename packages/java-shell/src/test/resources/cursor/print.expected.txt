[ { "0": { "_id": <ObjectID>, "a": 1 } }, { "0": { "_id": <ObjectID>, "a": 2 } }, { "0": { "_id": <ObjectID>, "a": 3 } } ]
[ { "0": "my document", "1": { "_id": <ObjectID>, "a": 1 } }, { "0": "my document", "1": { "_id": <ObjectID>, "a": 2 } }, { "0": "my document", "1": { "_id": <ObjectID>, "a": 3 } } ]
[ { "0": "Hello, world!" } ]